docs/blog/newsletter-april-2018.rst

```diff
@@ -0,0 +1,135 @@
+.. post:: April 9, 2018
+   :tags: newsletter
+
+
+########################################
+Write the Docs Newsletter – April 2018
+########################################
+
+Happy Monday! 
+Kelly O. here, with a ton of exciting community stuff to share with you this month! 
+There's a lot to cover, so I'll cut right to the chase!
+
+First, we're coming up on the Portland conference here in just four weeks (!?), and we're starting to get pretty stoked. 
+Tickets are now fully sold out, but if you're itching for a conference fix, now's a great time to look ahead to Prague in September... especially if you'd like to speak! 
```

Comment:
Tickets are fully sold out, but if you're itching for a conference fix, now's a great time to look ahead to Prague in September... especially if you'd like to speak! 

-- The two uses of "now" got a little confusing.